Bottom: 05bb3ef91175344d488c9c73b63f40068f9250d1
Top:    12aa5d356255440a8b5c150ee53770251e70861a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 16:14:27 +0000

Enable -Wextra (NOT yet -pedantic)


---

diff --git a/configure.ac b/configure.ac
index 058fa40..10d84b5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1143,7 +1143,7 @@ LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_L
 # if we have gcc then add -Wall
 if test "x$GCC" = "xyes"; then
 	# see about adding some extra checks if the compiler takes them
-	for flag in -Wall ; do
+	for flag in -Wall -Wextra; do #-pedantic
 		case " ${CFLAGS} " in
 			*\ ${flag}\ *)
 				# flag is already present
